NativeApp/src/components/User/View/index.js

```diff
@@ -6,57 +6,34 @@ import {
   FlatList,
 } from 'react-native';
 import ListItem from './ListItem';
-import { HeaderContainer, HolidayContainer, HolidayText } from './styled';
-import { ScrollView, H1, H2, P } from '../../Common';
-import colours from '../../../utilities/globalStyles/theme';
+import ListHeader from './ListHeader';
+import { ScrollView, H2, HeaderDays } from '../../Common';
 
 const UserView = (props) => {
   const { takenHolidays, remainingHolidays, employee } = props;
 
   return (
     <ScrollView>
-      <HeaderContainer>
-        <HolidayContainer divider>
-          <H2 type="base" styleProp={styles.boldDarkGrey}>
-            Taken
-          </H2>
-          <HolidayText>
-            <H1 type="base" styleProp={styles.darkGrey}>
-              {takenHolidays.length}
-            </H1>
-            <P type="base" styleProp={styles.darkGreyPadding}>
-              DAYS
-            </P>
-          </HolidayText>
-        </HolidayContainer>
-        <HolidayContainer>
-          <H2 type="base" styleProp={styles.boldDarkGrey}>
-            Remaining
-          </H2>
-          <HolidayText>
-            <H1 type="base" styleProp={styles.darkGrey}>
-              {remainingHolidays}
-            </H1>
-            <P type="base" styleProp={styles.darkGreyPadding}>
-              DAYS
-            </P>
-          </HolidayText>
-        </HolidayContainer>
-      </HeaderContainer>
-
-      <View style={styles.holsDate}>
-        <P type="base">
-          {employee.forename}
-          {employee.surname}
-        </P>
+    
+      <HeaderDays
+        takenHolidays={takenHolidays}
+        remainingHolidays={remainingHolidays}
+      />
+      <View style={styles.profileName}>
+        <H2 type="base" styleProp={styles.H2Bold}>
+          {employee.forename}&nbsp;
+          {employee.surname}'s Holidays
```

Comment:
You can do `{`${employee.surname}'s Holidays`}`. Looks cleaner.

Reply:
woops, completely forgot about template literals. yes, i have made the change